### ISSUE TITLE
 Download progress indicator within the app has too many decimal places 

### ISSUE THREAD

**Issue by maparr** [spacemeshos/smapp#1223]
### Describe the bug

Cosmetic: Download progress indicator within the app has too many decimal places 

### Steps to reproduce

When a new version is available, press 'Download'

### Expected behavior

App should display approximate progress such as 56% complete

### Actual behavior

3.0.0 and 3.0.1 display detailed progress: 56.327645273457623446725% Complete

**Comment by eros-fabbri** [comment]
is it still a relevant issue?

**Comment by brusherru** [comment]
Yes, it still just renders the `percent` number coming from `electron-autoupdater` handle as is (e.g. `56.32764527345762344672`) — need to round the number.
